apps/rush-lib/src/cli/scriptActions/BulkScriptAction.ts

```diff
@@ -329,6 +392,18 @@ export class BulkScriptAction extends BaseScriptAction {
         ' For details, refer to the website article "Selecting subsets of projects".'
     });
 
+    this._watchParameter = this.defineFlagParameter({
+      parameterLongName: '--watch',
+      parameterShortName: '-w',
+      description:
+        'Normally Rush would terminate after the command finishes;' +
+        ' adding this parameter will instead watch the file system for changes to the selected projects' +
+        ' (or all projects if no selection was specified).' +
```

Comment:
`rush build --watch` without any selectors should build all projects, but it seems to select nothing

Reply:
Fixed.